### PR TITLE
[RFR] Improve security

### PR DESCRIPTION
**Pull request by Kmaschta**
- [x] Limit user login by time [like this](https://github.com/marmelab/WMS/pull/66)

preview

### CODE REVIEW

**Comment by djhi** [comment]
:+1: BTW, IMO we should probably add a reasonable rate limiting to all api calls anyway.

**Comment by Kmaschta** [comment]
Good point, I add it to the TODO list.
Rebased

File: src/api/server.js [hunk 1]
```diff
@@ -137,13 +137,7 @@ app.use(function* (next) {
 });
 
 if (env !== 'development') {
-  // Hide powered-by koa
```
Comment (alexisjanvier) [inline src/api/server.js]: This is already removed by `koaHelmet` ? That's it ?

Comment (Kmaschta) [inline src/api/server.js]: Exactly!

File: src/api/users/authenticateRoutes.js [hunk 2]
```diff
@@ -5,10 +5,12 @@ import koa from 'koa';
 import koaRoute from 'koa-route';
 import methodFilter from '../lib/middlewares/methodFilter';
 import userRepositoryFactory from './userModel';
+import rateLimiter from './rateLimiter';
```
Comment (alexisjanvier) [inline src/api/users/authenticateRoutes.js]: @djhi I merge that before or after your # rebase #24 ?

**Comment by alexisjanvier** [comment]
:+1:

File: src/api/users/authenticateRoutes.js (hunk 2 above)
Comment (djhi) [inline src/api/users/authenticateRoutes.js]: merge it now, I'll rebase

**Pull request merged**